autoload/bufferline/highlight.vim

```diff
@@ -36,7 +36,7 @@ function bufferline#highlight#setup()
    \ ['BufferInactiveSign',   fg_subtle,   bg_inactive],
    \ ['BufferInactiveTarget', fg_target,   bg_inactive,  'bold'],
    \ ['BufferTabpages',       fg_special,  bg_inactive, 'bold'],
-   \ ['BufferTabpageSpacer',  fg_subtle,   bg_current],
+   \ ['BufferTabpageSpacer',  fg_subtle,   'NONE'],
```

Comment:
`'NONE'` will make the highlight group follow the background of whatever it is on. This way, we can at least provide an option for this space to be highlighted separately, while specifying that the default should be for it to be highlighted by the `*Inactive*` groups.

Reply:
LGTM, two things:
 - `'none'` lowercase as the other strings (`'bold'`)
 - s/Spacer/Fill/, to reuse vim semantics (`TabLineFill`)